lib/workos/client.rb

```diff
@@ -108,11 +108,15 @@ def handle_error_response(response:)
           request_id: response['x-request-id'],
         )
       when 422
-        errors = json['errors'].map do |error|
-          "#{error['field']}: #{error['code']}"
-        end.join('; ')
-
-        message = "#{json['message']} (#{errors})"
+        errors = nil
```

Comment:
Curious to hear context on what you encountered to need this change!

Also, you should be able to do

```rb
errors = if json['errors']
  json['errors'].map do |error|
    "#{error['field']}: #{error['code']}"
  end.join('; ')
end
```

This will put `nil` in `errors` if `json['errors']` is not present.

Reply:
The `send_session` error response only have a `message` key and no `errors` key so `json['errors'].map` was throwing an error. I was attempting to only run the map if `json['errors']` is present.